test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -865,3 +866,13 @@ TEST_VM(os, iso8601_time) {
   // Canary should still be intact
   EXPECT_EQ(buffer[os::iso8601_timestamp_size], 'X');
 }
+
+TEST_VM(os, is_first_C_frame) {
+  #ifndef _WIN32
```

Comment:
Spaces before `#` should ideally be avoided: https://stackoverflow.com/questions/4721978/should-preprocessor-instructions-be-on-the-beginning-of-a-line
(I guess not really a problem for the compilers we use.)